Python/pystate.c

```diff
@@ -2,6 +2,7 @@
 /* Thread and interpreter state structures and their interfaces */
 
 #include "Python.h"
+#include "objimpl.h"
```

Comment:
"Python.h" includes "objimpl.h" so I don't think you need to include it directly.

Reply:
I removed it (it was my editor that added it automatically)